marklogic/tests/test_parsers.py

```diff
@@ -0,0 +1,535 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think testing the parsers `test_parsers.py` against fixtures might not be needed since we already test against real MarkLogic service.

Reply:
I think it really helps understanding the logic of the check. It would be helpful for support and maintenance. Maybe these tests should be lightened, but I should we should keep them